content/essentials/stores.mdx

```diff
@@ -0,0 +1,203 @@
+While signals are created using the `createSignal` function and require calling the signal function to access their values, store values can be directly accessed without a function call.
+This provides access to the store's value directly within a tracking scope:
+
+```jsx
+const App = () => {
+  const [mySignal, setMySignal] = createSignal("This is a signal.")
+  const [store, setStore] = createStore({
+    userCount: 3,
+    users: [
+      {
+        id: 0,
+        username: 'felix909',
+        location: 'England',
+        loggedIn: false,
+      },
+      {
+        id: 1,
+        username: 'tracy634',
+        location: 'Canada',
+        loggedIn: true,
+      },
+      {
+        id: 2,
+        username: 'johny123',
+        location: 'India',
+        loggedIn: true,
+      }
+    ]
+  })
+  return (
+    <div>
+      <h1>Hello, {store.users[0].username}</h1> // Accessing a store value
+      <span>{mySignal()}</span> // Accessing a signal
+    </div>
+  );
+};
+```
+
+### Modifying store values
+
+Updating values within a store is best accomplished using a setter provided by the `createStore` initialization.
+This setter allows for the modification of a specific key and its associated value, following the format `setStore(key, newValue)`:
+
+```jsx
+setStore("users", (otherUsers) =>[ 
+    ...otherUsers,
+    {
+    id: 3,
+    username: 'michael584',
+    location: 'Nigeria',
+    loggedIn: false
+}])
+```
+
+Modifying a store using this method is referred to as "path syntax." 
+In this approach, the initial arguments are used to specify the keys that lead to the target value you want to modify, while the last argument provides the new value.
+
+<Aside>
+  Separating read and write capabilities to a store presents a valuable debugging advantage.
+  It facilitates the tracking and control of which components can access or change store values.
+</Aside>
+
+### Path syntax flexibilty 
+
+String keys are frequently used to precisely target particular values.
+By specifying these exact key names, you can direclty retrieve the desired information.
+However, path syntax goes beyond string keys and offers more versatility when accessing targeted values.
+
+Instead of employing the use of just single string keys, there is the option of using an array of keys.
+This method grants you the ability to select multiple properties within the store, facilitating accessed to nested structures.
+Alternatively, you can use filtering functios to access keys based on dynamic conditions or specific rules.
+
+[View on Eraser![](https://app.eraser.io/workspace/maDvFw5OryuPJOwSLyK9/preview?elements=aUOwffJd4QDWWDi22XI8DA&type=embed)](https://app.eraser.io/workspace/maDvFw5OryuPJOwSLyK9?elements=aUOwffJd4QDWWDi22XI8DA)
+
+The flexibility in path syntax makes for efficient navigation and data retrieval from your store, regardless of the store's complexity or the requirement for dynamic access scenarios within your application.
```

Comment:
"The flexibility in path syntax makes for efficient navigation and data modification from your store..."

I think retrieval should be modification because this is a pattern used in store setters